MzML/MzmlMethods.cs

```diff
@@ -820,7 +820,7 @@ public static void CreateAndWriteMyMzmlWithCalibratedSpectra(IMsDataFile<IMsData
                         // ONLY WRITING M/Z AND INTENSITY DATA, NOT THE CHARGE! (but can add charge info later)
                         // CHARGE (and other stuff) CAN BE IMPORTANT IN ML APPLICATIONS!!!!!
                         count = 5.ToString(),
-                        binaryDataArray = new Generated.BinaryDataArrayType[5]
+                        binaryDataArray = new Generated.BinaryDataArrayType[6]
```

Comment:
Would you please list the 6 arrays? Thank you!

Reply:
No problem. I commented them they are also here:
                        [0]: M/Z Data
                        [1]: Intensity Data
                        [2]: Sampled Noise Intensity
                        [3]: Sampled Noise M/Z (Mass)
                        [4]: Sampled Noise Baseline Intensity
                        [5]: Resolution Data